docs/modelserving/v1beta1/llm/huggingface/multi-node/README.md

```diff
@@ -0,0 +1,330 @@
+
+
+```yaml
+export HF_TEST_TOKEN=%token%
+export MODEL=meta-llama/Meta-Llama-3-8B-Instruct
+
+curl -o download-model-to-pvc.yaml https://kserve.github.io/website/latest/modelserving/v1beta1/llm/huggingface/multi-node/download-model-to-pvc.yaml
+envsubst < download-model-to-pvc.yaml | kubectl create -f -
+```
+
+
+### 3. Create a ServingRuntime
+
+Apply the ServingRuntime configuration. Replace %TBD% with the path to your ServingRuntime YAML.
+
+```bash
+kubectl apply -f https://github.com/kserve/kserve/blob/master/config/runtimes/kserve-huggingfaceserver-multinode.yaml
+```
+
+### 4. Deploy the model 
+
+Finally, deploy the model using the following InferenceService configuration:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  annotations:
+    serving.kserve.io/deploymentMode: RawDeployment
+    serving.kserve.io/autoscalerClass: external
+  name: huggingface-llama3
+spec:
+  predictor:
+    model:
+      runtime: kserve-huggingfaceserver-multinode
+      modelFormat:
+        name: huggingface
+      storageUri: pvc://llama-3-8b-pvc/hf/8b_instruction_tuned  
+    workerSpec: {}
+EOF
+```      
+
+## Check `InferenceService` status.
+
+To verify the status of your `InferenceService`, run the following command:
+
+```bash
+kubectl get inferenceservices huggingface-llama3
+```
+
+!!! success "Expected Output"
+    ```{ .bash .no-copy }
+    NAME                 URL                                                   READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION                          AGE
+    huggingface-llama3   http://huggingface-llama3.default.example.com                                          5m
+    ```
+
+## Check `GPU resource` status.
+
+To check the GPU resource status, follow these steps:
+
+ 1. Retrieve the pod names for the head and worker nodes:
+    ```bash
+    # Get pod name
+    podName=$(kubectl get pod -l app=isvc.huggingface-llama3-predictor --no-headers|cut -d' ' -f1)
+    workerPodName=$(kubectl get pod -l app=isvc.huggingface-llama3-predictor-worker --no-headers|cut -d' ' -f1)
+    ```
+
+  2. Check the GPU memory size for both the head and worker pods:
+      ```bash
+      # Check GPU memory size
+      kubectl exec $podName -- nvidia-smi
+      kubectl exec $workerPodName -- nvidia-smi
+      ```
+
+!!! success "Expected Output"
+    ```{ .bash .no-copy }
+    +-----------------------------------------------------------------------------------------+
+    | NVIDIA-SMI 550.90.07              Driver Version: 550.90.07      CUDA Version: 12.4     |
+    |-----------------------------------------+------------------------+----------------------+
+    | GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
+    | Fan  Temp   Perf          Pwr:Usage/Cap |            # Specifying workerSpec indicates that multi-node functionality will be used     Memory-Usage | GPU-Util  Compute M. |
```

Comment:
There's some unexpected copy paste here

Reply:
oh.. yes.. my new mouse made this kind of unexpected copy/paste thing.
removed.